src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -833,4 +833,23 @@ private static void addExecutablePermissionToDirectory(String filePath) {
                     + "'");
         }
     }
+
+    /**
+     * Update the tool if any latest version available.
+     *
+     * @param printStream stream which messages should be printed
+     */
+    public static void updateTool(PrintStream printStream) {
+        String version = ToolUtil.getCurrentToolsVersion();
+        printStream.println("checking whether any latest tool version is available...");
+        String latestVersion = ToolUtil.getLatestToolVersion();
+        if (latestVersion == null) {
+            printStream.println("Failed to find the latest tool version");
+            return;
+        }
+        if (latestVersion.equals(version)) {
+            return;
+        }
+        ToolUtil.downloadTool(printStream, latestVersion);
```

Comment:
Shouldn't we use the word `update tool` or `ballerina update tool` in all the user logs, instead of using just `tool`?